### PR TITLE
Update 100.13.0

### PR DESCRIPTION
**Pull request by zkline101**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -32,14 +32,19 @@ let package = Package(
...
-            targets: ["ArcGIS"]
+            targets: ["ArcGIS", "Runtimecore"]
         )
     ],
     targets: [
         .binaryTarget(
             name: "ArcGIS",
-            url: "https://developers.arcgis.com/ios/downloads/ArcGIS-iOS-v100.12.xcframework.zip",
-            checksum: "15b34f66dec01f1203479a91600a0b81ecb6790ae18b3aa69c8de9f814f506ad"
+            url: "https://developers.arcgis.com/ios/downloads/ArcGIS-iOS-v100.13.xcframework.zip?dl=1",
```
Comment (dg0yal) [inline Package.swift]: Do we really need the `?dl=1` parameter in the URL?

File: Package.swift [hunk 2]
```diff
@@ -32,14 +32,19 @@ let package = Package(
...
+            checksum: "98231f876f18954fe5426750b2910002f6942a3d6d8fff862abcdff17525f2eb"
+        ),
+        .binaryTarget(
+            name: "Runtimecore",
+            url: "https://developers.arcgis.com/ios/downloads/Runtimecore-v100.13.xcframework.zip?dl=1",
```
Comment (dg0yal) [inline Package.swift]: Do we really need the `?dl=1` parameter in the URL?

File: Package.swift (hunk 1 above)
Comment (zkline101) [reply Package.swift]: Whoops your right we don't need that it was from another link I was using to test

**Review by dg0yal — APPROVED** [review]

**Comment by zkline101** [comment]
Thanks!

**Pull request merged**